src/ast/ast_validations.c

```diff
@@ -2381,6 +2381,15 @@ AST_Validation AST_Validate_Query
 	ast.root = body;
 
 	cypher_astnode_type_t body_type = cypher_astnode_type(body);
+
+	if(body_type == CYPHER_AST_CREATE_NODE_PROP_CONSTRAINT ||
+	   body_type == CYPHER_AST_CREATE_REL_PROP_CONSTRAINT ||
+	   body_type == CYPHER_AST_DROP_NODE_PROP_CONSTRAINT ||
+	   body_type == CYPHER_AST_DROP_REL_PROP_CONSTRAINT) {
+		ErrorCtx_SetError(EMSG_INVALID_CONSTRAINT_COMMAND);
+		return AST_INVALID;
+	}
```

Comment:
Seems like the parser supports these clauses, do you recall why did we decided on introducing an actual command instead of managing constraints using `GRAPH.QUERY` ?

Reply:
so we can see the usage of the GRAPH.CONSTRAINT command and use ACL
this is not really a cypher query

Reply:
understood!